internal/runners/initialize/init.go

```diff
@@ -78,10 +78,13 @@ var errNoOwner = errs.New("Could not find organization")
 var errNoLanguage = errs.New("No language specified")
 
 type errUnrecognizedLanguage struct {
-	error
 	Name string
 }
 
+func (e errUnrecognizedLanguage) Error() string {
+	return fmt.Sprintf("unrecognized language: %s", e.Name)
```

Comment:
Does this error message get printed or does the user-facing error handle this?

Reply:
The user-facing error handles this. This message only shows up in the debug logs.